src/PJ_isea.c

```diff
@@ -3,15 +3,49 @@
  * and is in the public domain.
  */
 
+#include <errno.h>
 #include <float.h>
 #include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
-#ifndef M_PI
-#  define M_PI 3.14159265358979323846
-#endif
+#define PJ_LIB__
+#include "proj.h"
+#include "projects.h"
+
+#define DEG36 0.62831853071795864768
+#define DEG72 1.25663706143591729537
+#define DEG90 M_PI_2
+#define DEG108 1.88495559215387594306
+#define DEG120 2.09439510239319549229
+#define DEG144 2.51327412287183459075
+#define DEG180 M_PI
+
+/* sqrt(5)/M_PI */
+#define ISEA_SCALE 0.8301572857837594396028083
+
+/* 26.565051177 degrees */
+#define V_LAT 0.46364760899944494524
+
+/* 52.62263186 */
+#define E_RAD 0.91843818702186776133
+
+/* 10.81231696 */
+#define F_RAD 0.18871053072122403508
+
+/* R tan(g) sin(60) */
+#define TABLE_G 0.6615845383
+
+/* H = 0.25 R tan g = */
+#define TABLE_H 0.1909830056
+
+/* in radians */
+#define ISEA_STD_LAT 1.01722196792335072101
+#define ISEA_STD_LON .19634954084936207740
+
+#define RAD2DEG (180.0/M_PI)
```

Comment:
Since you've already started the clean up: `proj_internal.h` provides `PJ_TODEG` and `PJ_TORAD`. Use those and you can more or less close https://github.com/OSGeo/proj.4/issues/991.

Reply:
PJ_TO* done in 8d42763.  Still have proj_todeg and PJ_TODEG, so #991 isn't quite done yet.

Reply:
That's by design. `proj_todeg` is primarily there for API users and `PJ_TODEG` is for internal use only (saving a few function calls by using the preprocessor).